packages/secure_storage/amplify_secure_storage_test/lib/amplify_secure_storage_test.dart

```diff
@@ -148,31 +155,26 @@ void runStandardTests(
     });
   });
 
-  group('write', () {
-    test('writes a new key-value pair to storage', () async {
-      // write value
-      await storage.write(key: key1, value: 'test_write');
-
-      // confirm value was written
-      final value1 = await storage.read(key: key1);
-      expect(value1, 'test_write');
-    });
-
-    test('updates the value for an existing key', () async {
-      // write value
-      await storage.write(key: key1, value: 'test_write');
-
-      // confirm value was written
-      final value1 = await storage.read(key: key1);
-      expect(value1, 'test_write');
-
-      // write new value to the existing key
-      await storage.write(key: key1, value: 'test_update');
-
-      // confirm new value was written
-      final value2 = await storage.read(key: key1);
-      expect(value2, 'test_update');
-    });
+  group('read/write/delete can handle key value pairs of varying length', () {
+    for (var entry in keyValuePairs.entries) {
```

Comment:
```suggestion
    for (final entry in keyValuePairs.entries) {
```

Don't necessarily need to change since I don't care as much about the style of test code but I thought this was a lint